### PR TITLE
feat: use supportFile by testingType

### PR DESCRIPTION
**Pull request by estrada9166**
- Closes  [UNIFY-662](https://cypress-io.atlassian.net/jira/software/projects/UNIFY/boards/20?assignee=5ea883deddd45f0b94f5d98d&selectedIssue=UNIFY-662)

### PR Tasks

- [ ] Have tests been added/updated?
- [ ] Has the original issue (or this PR, if no issue exists) been tagged with a release in ZenHub? (user-facing changes only)
- [ ] Has a PR for user-facing changes been opened in [`cypress-documentation`](https://github.com/cypress-io/cypress-documentation)? 
- [ ] Have API changes been updated in the [`type definitions`](https://github.com/cypress-io/cypress/blob/develop/cli/types/cypress.d.ts)?
- [ ] Have new configuration options been added to the [`cypress.schema.json`](https://github.com/cypress-io/cypress/blob/develop/cli/schema/cypress.schema.json)?

### CODE REVIEW

File: .eslintignore [hunk 1]
```diff
@@ -22,7 +22,8 @@ system-tests/projects/**/static/*
 system-tests/projects/**/*.jsx
 system-tests/projects/**/fail.js
 system-tests/lib/scaffold/plugins/index.js
-system-tests/lib/scaffold/support/index.js
+system-tests/lib/scaffold/support/e2e.js
+system-tests/lib/scaffold/support/commaponent.js
```
Comment (lmiller1990) [inline .eslintignore:26]: Is this supposed to be spelled `commaponent.js`?

**Review by lmiller1990 — APPROVED** [review]
One minor thing, I went through this PR as carefully as I could, looks good!

**Comment by lmiller1990** [comment]
I tried it out, seems great! I'll use your new error handling stuff for the `specPattern` changes once this lands.

image

**Review by lmiller1990 — DISMISSED** [review]

File: packages/data-context/src/actions/WizardActions.ts [hunk 2]
```diff
@@ -311,16 +316,23 @@ export class WizardActions {
...
-  }
-`
+interface E2eScaffoldOpts {
+  lang: CodeLanguageEnum
+}
+
+const E2E_SCAFFOLD_BODY = (opts: E2eScaffoldOpts) => {
+  return `
+    e2e: {
+      supportFile: ${opts.lang === 'ts' ? '\'cypress/support/e2e.ts\'' : '\'cypress/support/e2e.js\''},
```
Comment (tgriesser) [inline packages/data-context/src/actions/WizardActions.ts:326]:
```suggestion
      supportFile: '\'cypress/support/e2e.${opts.lang}\'',
```

**Review by tgriesser — APPROVED** [review]

**Review by tgriesser — APPROVED** [review]

**Review by tgriesser — APPROVED** [review]

**Pull request merged**